### DISCUSSION TITLE
1.12.0 🎄

### DISCUSSION THREAD

**Discussion by jschlyter** [pypolestar/polestar_api#278] [Release Discussion]
Yesterday I made a [pre-release of 1.12.0](https://github.com/pypolestar/polestar_api/releases/tag/1.12.0) that might be useful for those with multiple cars and/or accounts. Please report back here if you've installed and tested.

**Comment by BJReplay** [comment]
Successfully installed and running, but only a single car / single account.

**Comment by jschlyter** [comment]
1.12.0 is now released 🎅
